pkg/adhoc/server/server.go

```diff
@@ -0,0 +1,119 @@
+package server
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+
+	"github.com/pyroscope-io/pyroscope/pkg/adhoc/util"
+)
+
+type profile struct {
+	ID        string    `json:"id"`
+	Name      string    `json:"name"`
+	UpdatedAt time.Time `json:"updated_at"`
```

Comment:
```suggestion
	UpdatedAt time.Time `json:"updatedAt"`
```

Based on the other fields I believe we are a camelCase team :laughing: